tools/ci_build/github/azure-pipelines/templates/download-deps.yml

```diff
@@ -24,5 +24,4 @@ steps:
       definition: 'fd9dd5ad-b73e-4678-890e-edcf680dbc1a'
```

Comment:
You may exclude this file from your PR.

Reply:
If you don't change yml files, you don't need to wait an approval from "@onnxruntime-es"

Reply:
I changed a lot these files until float 8 was part of onnx. I removed the empty line.